scripts/docker/README.md

```diff
@@ -45,6 +46,41 @@ export STAGING_DIR=~/tmp/tmp_build_docker_image/
 sudo bash build_docker.sh -e ${GITHUB_TAG} -p -d ${STAGING_DIR}
 ```
 
+#### Create GATK4 docker image and upload into GCR
+
+This will attempt to build and automatically upload the docker image with a tag following this format: `us.gcr.io/broad-dsde-methods/broad-gatk-snapshots/gatk-remote-builds:<USERNAME>-<GITHUB_TAG>-<GIT_HASH>`
+
+From this directory, run:
+
+```bash
+# REPLACE VALUE OF GITHUB_TAG WITH DESIRED VERSION
+export GITHUB_TAG=4.2.6.1
+
+# REPLACE with the directory where you'd like to clone the repo
+export STAGING_DIR=~/tmp/tmp_build_docker_image/
```

Comment:
These don't actually have to be exported, since you're passing them into the script directly below.

Reply:
I'm simply copying the syntax of the other dozen of these things in this folder